clients/filter/filter.go

```diff
@@ -510,7 +551,7 @@ func (c *Client) GetPreview(ctx context.Context, filterOutputID string) (p Previ
 	if err != nil {
 		return
 	}
-	defer resp.Body.Close()
+	defer CloseResponseBody(ctx, resp)
```

Comment:
This needs to come immediately after 
```
resp, err := c.cli.Do(ctx, req)
if err != nil {
    return
}
```

Reply:
The token stuff is all fine but there are a few methods that are either not deferring the resp.Body.Close call at all or they are not doing immediately after checking the resp err.